css/components/tile.css

```diff
@@ -40,9 +34,8 @@
   padding-top: 2vh;
 }
 
-.tile__p {
```

Comment:
Usually when using BEM or adding CSS classes it is not common to cite the HTML element. Would be better to name it according to the content.